wallet/wallet-controller/src/lib.rs

```diff
@@ -618,6 +647,31 @@ impl<T: NodeInterface + Clone + Send + Sync + 'static, W: WalletEvents> Controll
             }
 
             tokio::time::sleep(NORMAL_DELAY).await;
+
+            self.rebroadcast_txs(&mut rebroadcast_txs_timer);
+        }
+    }
+
+    /// Rebroadcast not confirmed transactions
+    fn rebroadcast_txs(&mut self, rebroadcast_txs_timer: &mut Instant) {
```

Comment:
We should avoid using Instant as it has issues with out time. Please try to use a regular clock and calculate differences... as dumb as it sounds. We should be able to mock time with no issues.

Reply:
Changed to use get_time()